### DISCUSSION TITLE
Planned Corpora

### DISCUSSION THREAD

**Discussion by Ronsor** [RyokoAI/BigKnow2022#4] [Announcements]
**List moved to README.md**

**Comment by Ristellise** [comment]
> Syosetu ni Narou (syosetu.com)

Should we include other english webnovels like those from scribblehub?

**Comment by Ristellise** [comment]
Gonna mention 2 additional sites that Ronsor and I talked about:
 - fc2 (Blog hosting website for Japan. Inital glance appears to be a lot of spam (for those with large blogs) but maybe we can filter out those? Scrape in progress)
 - atwiki (Wiki hosting website for Japan. Seems to contain a lot of useful infomation on games among other other topics like anime franchises. Possibly detailed game information as well? Fast to scrape too. Scraping currently)
